doc-content/enterprise-only/openshift/secrets-smartrouter-create-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='secrets-smartrouter-create-proc']
+= Creating the secrets for Smart Router
+
+If you are planning to deploy Smart Router in your OpenShift environment, 
+ifeval::["{context}"!="openshift-ansible-playbook"]
+you must create an SSL certificate for Smart Router and provide it to your OpenShift environment as a secret.
+endif::[]
+ifeval::["{context}"=="openshift-ansible-playbook"]
+this component uses an SSL certificate to provide HTTPS access. The deployment can create a sample secret automatically. However, in production environments you must create an SSL certificate for Smart Router and provide it to your OpenShift environment as a secret.
```

Comment:
Should the second ifeval say "note that this ..."? Sounds better and more purposeful to me than "If you are planning to deploy ... environment, this component ...". Not sure what the point of the sentence is. Saying "note that this" makes the point somewhat clear, to me.